physics/satmedmfvdifq.F

```diff
@@ -1336,7 +1360,8 @@ subroutine satmedmfvdifq_run(im,km,ntrac,ntcw,ntrw,               &
             ptem2 = ptem2 + ptem
 !
 !           tem2 = stress(i)*spd1(i)/zl(i,1)
-            tem2 = stress(i)*ustar(i)*phim(i)/(vk*zl(i,1))
+!           tem2 = stress(i)*ustar(i)*phim(i)/(vk*zl(i,1))
```

Comment:
Consider removing commented code. Old versions will always be saved in git history.